docs/crud/introduction.md

```diff
@@ -55,6 +55,10 @@ type Subscription {
 // highlight-end
 ```
 
+:::caution
```

Comment:
Did not know about this elements. Renders really nice in web (not github) THX

Reply:
Yep, Docusaurus have some noce elements

https://v2.docusaurus.io/docs/markdown-features/#calloutsadmonitions